compat/package.json

```
@@ -44,6 +44,10 @@
 		"./scheduler": {
 			"import": "./scheduler.mjs",
 			"require": "./scheduler.js"
+		},
+		"./package.json": {
+			"import": "./package.json",
+			"require": "./package.json"
 		}
```

This can be simplified a bit, since the formats do not differ. See the main `package.json` file:

https://github.com/preactjs/preact/blob/284a8b0d969f8ef3a3eeff0107b0104a52f774ad/package.json#L89

```suggestion
		"./package.json": "./package.json"
```

---

Thanks, makes sense!